bahmni_api_feed/models/order_save_service.py

```diff
@@ -393,7 +392,7 @@ def _create_sale_order_line_function(self, sale_order, order):
             product_uom_qty = order['quantity']
             if(prod_lot != None and order['quantity'] > prod_lot.stock_forecast and prod_lot.stock_forecast > 0):
                 product_uom_qty = prod_lot.stock_forecast
-            description = " ".join([prod_obj.name, "- Total", str(product_uom_qty), str(order.get('quantityUnits', None))])
+            description = " ".join([str(actual_quantity), str(order.get('quantityUnits', None))])
```

Comment:
In OpenERP, the description used to be concatenation of the (Original Product + Qty). Here it seems to be only (Qty). I think putting the Original Product prescribed might be more accurate, since people can change the product (from `PCM 10mg tablet` 15 tablets) to (`PCM 5mg tablet` 30 tablets) while dispensing, because they didn't have 10mg tablets in stock.

Reply:
In OpenERP, there is only units, the product name is added in Odoo 10. 
<img width="913" alt="image" src="https://github.com/Bahmni/bahmni-odoo-modules/assets/31698165/dc66cf4d-aae7-49c5-9188-e1dd54225e67">

Reply:
Updated with product name
<img width="1101" alt="image" src="https://github.com/Bahmni/bahmni-odoo-modules/assets/31698165/80d31b8f-81d6-4f3c-81c7-6ff1c9f3d9f1">
This is needed for print as well. When a quotation is printed, the description field only comes up